src/test/java/com/thealgorithms/maths/MobiusFunctionTest.java

```diff
@@ -0,0 +1,57 @@
+package com.thealgorithms.maths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class MobiusFunctionTest {
+
+	@Test
+	void testMobiusForZero() {
+		//given
+		int number = 0;
+		int expectedValue = 1;
+
+		//when
+		int actualValue = MobiusFunction.mobius(number);
+
+		//then
+		assertEquals(expectedValue, actualValue);
+	}
+
+	@Test
+	void testMobiusForNegativeNumber() {
+		//given
+		int number = -1;
+		int expectedValue = 1;
+
+		//when
+		int actualValue = MobiusFunction.mobius(number);
+
+		//then
+		assertEquals(expectedValue, actualValue);
+	}
```

Comment:
As far as I can tell, the Moebius function is undefined for negative integers and zero

Reply:
Fixed this [here](https://github.com/TheAlgorithms/Java/pull/3241/commits/bf9d3571477c85eea5024e7080c3d8f4f4955da7),  throwing IllegalArgumentException when method param is less than or equal to zero.